### PR TITLE
Simplify chatter data model #9

### PR DESCRIPTION
**Pull request by warpech**
Creating this pull request on behalf of @aloscha 

This is in order to finish the task https://github.com/StarcounterSamples/Chatter/issues/9

Code review for changes in Simplified repo will follow in this PR

### CODE REVIEW

File: Ring6/Chat/ChatAttachment.cs [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
 
-namespace Simplified.Ring6 {
-    public class ChatAttachment : Simplified.Ring1.Relation {
-        [SynonymousTo("ToWhat")]
-        public ChatMessage Message;
-
-        [SynonymousTo("WhatIs")]
-        public Simplified.Ring1.Something Attachment;
+namespace Simplified.Ring6
+{
```
Comment (warpech) [inline Ring6/Chat/ChatAttachment.cs]: Please make sure that this PR follows the coding style explained at: https://github.com/StarcounterPrefabs/Guidelines (regarding the position of `{`)

Comment (aloscha) [inline Ring6/Chat/ChatAttachment.cs]: :(

File: .vs/config/applicationhost.config [hunk 2]
```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joozek78) [inline .vs/config/applicationhost.config]: Is this file necessary? It looks like left-overs

Comment (warpech) [inline .vs/config/applicationhost.config]: Yeah, I have no idea what does it do. @aloscha could you pls review if it belongs to the repo or not? If not then pls add it to .gitignore

**Comment by per-samuelsson** [comment]
> Is this file necessary? It looks like left-overs

This file is something VS has started to constantly auto-generate for some reason. I always delete it. I think we should have it in all our .gitignore's until we see we really need it. Please don't push it until you are certain of why.

File: Ring6/Chat/TextRelation.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using Simplified.Ring1;
+using Starcounter;
+
+namespace Simplified.Ring6
+{
+    public class TextRelation : Relation
```
Comment (warpech) [inline Ring6/Chat/TextRelation.cs]: Following the [convention](https://github.com/StarcounterSamples/Simplified/blob/master/Ring3/Address/AddressRelation.cs) used in other classes in this repo, this class should be called `ChatMessageTextRelation`. Following another [convention](), it could be called `ChatMessagesChatMessageText`.

But the current name `TextRelation` is definitively too simple.

File: Ring6/Chat/TextRelation.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    {
+        [SynonymousTo("WhatIs")]
+        public Something Concept;
+
+        [SynonymousTo("ToWhat")]
+        public ChatMessageText Content;
+    }
+}
```
Comment (warpech) [inline Ring6/Chat/TextRelation.cs]: Here you inverted the relation. I think that it should be:

```cs
[SynonymousTo("WhatIs")]
public ChatMessageText ChatMessageText;
 
[SynonymousTo("ToWhat")]
 public ChatMessage ChatMessage;
```

File: Ring1/Illustration.cs [hunk 5]
```diff
@@ -1,6 +1,12 @@
...
+    public class Illustration : Relation
+    {
+        [SynonymousTo("WhatIs")]
         public Something Concept;
+
+        [SynonymousTo("ToWhat")]
         public Content Content;
     }
-}
+}
```
Comment (warpech) [inline Ring1/Illustration.cs]: Here you inverted the relation. I think that it should be:

```cs
[SynonymousTo("WhatIs")]
public Something Concept;
 
[SynonymousTo("ToWhat")]
public Content Content;
```

Read this in plain English: A Content is an Illustration of a Concept.
More generally: A .jpg is an illustration of a Something.

**Comment by warpech** [comment]
I presented some of my comments. Please review them and apply. I will continue the review tomorrow.

File: .vs/config/applicationhost.config (hunk 2 above)
Comment (aloscha) [inline .vs/config/applicationhost.config]: Fixed, I add it to .gitignore and remove it from project

File: Ring6/Chat/ChatAttachment.cs (hunk 1 above)
Comment (aloscha) [inline Ring6/Chat/ChatAttachment.cs]: Fixed in all classes, added on that branch

File: Ring6/Chat/TextRelation.cs (hunk 3 above)
Comment (aloscha) [inline Ring6/Chat/TextRelation.cs]: Fixed, name changed to ChatMessageTextRelation

File: Ring1/Illustration.cs (hunk 5 above)
Comment (aloscha) [inline Ring1/Illustration.cs]: I code we have exactly like you write above.

File: Ring6/Chat/TextRelation.cs (hunk 4 above)
Comment (aloscha) [inline Ring6/Chat/TextRelation.cs]: I think each ChatMessage could have ChatMessage, Also We ccouldn't say that ChatMessage is ToWhat, more preferably will be Something

File: Ring1/Illustration.cs (hunk 5 above)
Comment (warpech) [inline Ring1/Illustration.cs]: Sorry, that was a typo. I meant: 

```cs
[SynonymousTo("WhatIs")]
public Content Content;

[SynonymousTo("ToWhat")]
public Something Concept;
```

Read this in plain English: A Content is an Illustration of a Concept.
More generally: A .jpg is an illustration of a Something.

File: Ring6/Chat/TextRelation.cs (hunk 4 above)
Comment (warpech) [inline Ring6/Chat/TextRelation.cs]: Fine, but then anyway SynonymousTo is inverted. It should be:

```cs
[SynonymousTo("WhatIs")]
public ChatMessageText ChatMessageText;

[SynonymousTo("ToWhat")]
public Something Concept;
```

Comment (aloscha) [inline Ring6/Chat/TextRelation.cs]: Fixed in new commits

File: Ring1/Illustration.cs (hunk 5 above)
Comment (aloscha) [inline Ring1/Illustration.cs]: Fixed in new commits
